backend/src/agents/agent.py

```diff
@@ -37,7 +38,7 @@ def __get_action(self, utterance: str) -> Action_and_args:
 
         logger.debug(f"List of tools: {tool_descriptions}")
 
-        response = json.loads(self.llm.chat(format_prompt, tools_available))
+        response = json.loads(self.llm.chat("mistral-large-latest", format_prompt, tools_available))

```

Comment:
mistral model is hard coded here. It should come from the config.

Reply:
Yes I hardcoded it here because the agent class doesn't have a default model, but I appreciate it's not ideal. I'll create an agent class model env variable, unless you have another idea

Reply:
I think we need to create an env variable otherwise once we start using openai or any other LLM in combination with this agent.py it may fail as I am not sure how it's going to work in combination with any other LLM.